Code_Exercises/Exercise_05_Device_Selection/solution.cpp

```diff
@@ -59,15 +59,10 @@ TEST_CASE("intel_gpu_device_selector", "device_selectors_solution") {
   int a = 18, b = 24, r = 0;
 
   try {
-    auto asyncHandler = [&](sycl::exception_list exceptionList) {
```

Comment:
I think the pull request  has some other commits in it, maybe it needs to be rebased?